test/hotspot/jtreg/compiler/arraycopy/TestObjectArrayClone.java

```diff
@@ -26,7 +26,11 @@
  * @bug 8155643
  * @summary Test Object.clone() intrinsic if ReduceInitialCardMarks is disabled.
  *
- * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions -Xcomp -XX:-ReduceInitialCardMarks
+ * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions -XX:-ReduceInitialCardMarks
+ *                   -XX:CompileCommand=compileonly,compiler.arraycopy.TestObjectArrayClone::test
+ *                   compiler.arraycopy.TestObjectArrayClone
+ *
+ * @run main/othervm -XX:+IgnoreUnrecognizedVMOptions -XX:+UseZGC
```

Comment:
I suggest to clone it to separate `@test` block because you need `@requires  vm.gc.Z` for it.

Reply:
I removed the explicit GC. I should be run in all configs.